docs/protocol/off-chain/queries.mdx

```diff
@@ -0,0 +1,801 @@
+| `get_session_keys` | Private | List active session keys | No |
+| `ping` | Public | Connection health check | No |
+
+:::info Authentication
+**Public methods** can be called without authentication. **Private methods** require completing the [authentication flow](./authentication) first.
+:::
+
+---
+
+## get_config
+
+### Name
+
+`get_config`
+
+### Usage
+
+Retrieves <Tooltip content="A virtual ledger layer that provides a unified ledger (through Nitro RPC) and coordinates state channels (through Nitrolite), providing chain abstraction for developers and users">the clearnode's</Tooltip> configuration including supported blockchains, protocol versions, challenge periods, and operational parameters.
+
+### Request
+
+No parameters required (empty object `{}`).
+
+### Response
+
+| Parameter | Type | Description | Structure | Example |
+|-----------|------|-------------|-----------|---------|
+| `broker_address` | string | Clearnode's primary address | - | `"0xClearnode123..."` |
+| `supported_chains` | array\<ChainConfig\> | List of supported blockchain networks<br/><br/>**Structure per chain**:<br/>• `chain_id` (number) - Network identifier (e.g., 1 for Ethereum)<br/>• `name` (string) - Network name<br/>• `rpc_url` (string) - RPC endpoint<br/>• `contracts` (object) - Deployed contract addresses | See structure | - |
+| `protocol_versions` | array\<string\> | Supported Nitro RPC versions | - | `["NitroRPC/0.2", "NitroRPC/0.4"]` |
+| `features` | object | Enabled features and capabilities | - | `{"app_sessions": true, "cross_chain": true, "max_session_duration": 86400}` |
+
+**Use Cases**:
+- Verify clearnode capabilities before connecting
+- Discover supported chains and assets
+- Check protocol version compatibility
+
+---
+
+## get_assets
+
+### Name
+
+`get_assets`
+
+### Usage
+
+Retrieves all supported assets and their configurations across supported blockchains.
+
+### Request
+
+| Parameter | Type | Required | Description | Example | Notes |
+|-----------|------|----------|-------------|---------|-------|
+| `chain_id` | uint64 | No | Filter by specific chain | `137` | If omitted, returns assets for all chains |
+
+### Response
+
+| Parameter | Type | Description | Structure | Example |
+|-----------|------|-------------|-----------|---------|
+| `assets` | array\<AssetConfig\> | List of supported assets<br/><br/>**Structure per asset**:<br/>• `symbol` (string) - Asset symbol<br/>• `name` (string) - Full asset name<br/>• `decimals` (uint8) - Token decimals<br/>• `chains` (array\<ChainDeployment\>) - Deployed addresses per chain<br/>&nbsp;&nbsp;- Structure per deployment:<br/>&nbsp;&nbsp;&nbsp;&nbsp;▪ `chain_id` (number) - Network identifier<br/>&nbsp;&nbsp;&nbsp;&nbsp;▪ `token_address` (string) - Contract address on this chain | See structure | symbol: `"USDC"`<br/>name: `"USD Coin"`<br/>decimals: `6`<br/>chains: `[{"chain_id": 1, "token_address": "0xA0b8..."}, {"chain_id": 137, "token_address": "0x2791..."}]` |
```

Comment:
Asset has the following structure:
https://github.com/erc7824/nitrolite/blob/c708013f7bb2436f2ff7112be5e58f2d6058d79d/clearnode/pkg/rpc/api.go#L560-L569